Library/Homebrew/formula.rb

```diff
@@ -1162,9 +1164,10 @@ def patch
   # @private
   def brew
     @prefix_returns_versioned_prefix = true
+    active_spec.fetch
```

Comment:
So I can follow: in what scenarios is this needed? This can cause double fetching (albeit with caching in the second run).

Reply:
It's been made conditional. It's mainly needed for tests.